engine/link/link.asm

```diff
@@ -73,6 +73,7 @@ if DEF(_CRYSTALVC)
 else
 	ld c, 3
 endc
+	vc_patch_end NetworkDelay1
```

Comment:
You made a `vc_patch_end` macro that doesn't take any arguments (you're using `CURRENT_VC_PATCH` to specify the label name, not `\1`), so the argument on each of these is redundant.

Reply:
Removed the redundant argument in https://github.com/pret/pokecrystal/pull/882/commits/c6c7051039a706c950e4e883fd3acc8cf1bc5469